src/PhpSpreadsheet/Reader/Xls.php

```diff
@@ -640,7 +648,7 @@ public function load($pFilename)
 
         // initialize
         $this->pos = 0;
-        $this->codepage = 'CP1252';
+        $this->codepage = $this->codepage == null ? 'CP1252' : $this->codepage;
```

Comment:
Perhaps use `$this->codepage = ?: 'CP1252'` here.
As our minimum PHP version is 7.1, using more succinct syntax is available to us

Reply:
done